src/presenters/personal/personal-name-check-presenter.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Formats data ready for presenting in the `/personal-name-check` page
+ * @module personalNameCheckPresenter
+ */
+
+const personalNameCheckPresenter = (personalDetails) => {
+  return {
+    backLink: { href: '/account-name-change' },
+    changeLink: '/account-name-change',
+    pageTitle: 'Check your name is correct before submitting',
+    metaDescription: 'Check the full name for your personal account is correct.',
+    userName: personalDetails.info.fullName.fullNameJoined ?? null,
+    fullName: formatFullName(personalDetails.changePersonalName ?? personalDetails.info.fullName)
+  }
+}
+
+/**
+ * Formats the full name (first, middle and last) into a single string
+ */
+const formatFullName = (nameData) => {
```

Comment:
Ive seen this function in multiple places. Can we lift it into the base presenter?